lang-python/xonsh/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=xonsh
+PKGSEC=shells
+PKGDEP="pygments prompt-toolkit py-setproctitle"
+PKGDES="Python-powered shell. Full-featured and cross-platform."
+
```

Comment:
You will also need to create an entry in `/etc/shells`, see `app-shells/fish/autobuild/postinst`.